packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -54,58 +54,59 @@ const componentName: string = "stark-table";
 /* tslint:enable */
 export class StarkTableComponent extends AbstractStarkUiComponent implements OnInit, AfterContentInit, AfterViewInit, OnChanges {
 	/**
-	 * HTML id of the table
+	 * Array of StarkColumnProperties objects which define the columns of the data table.
 	 */
 	@Input()
-	public htmlId: string;
+	public columnProperties: StarkTableColumnProperties[] = [];
 
 	/**
-	 * Data that will be display inside your table.
+	 * Array of StarkAction objects (see StarkAction docs).
 	 */
 	@Input()
-	public data: any[];
+	public customTableActions?: StarkAction[];
 
 	/**
-	 * Object which contains filtering information for the table.
+	 * Type of StarkAction objects
```

Comment:
I suggest we change this description like this: "Mode in which the custom actions will be displayed in the table".
What do you think?

Reply:
I think it seems ok. I change that too